addons/pos_restaurant/tests/test_frontend.py

```diff
@@ -221,3 +222,12 @@ def test_07_split_bill_screen(self):
     def test_08_refund_stay_current_table(self):
         self.pos_config.with_user(self.env.ref('base.user_demo')).open_ui()
         self.start_tour("/pos/ui?config_id=%d" % self.pos_config.id, 'RefundStayCurrentTableTour', login="demo")
+
+    def test_09_tips_after_payment_format_currency(self):
```

Comment:
It would indeed be nice to test that we are handling the decimal point correctly, but i think that we should have a generic test checking that the currency is handled correctly. Testing this in the restaurant module seems misleading, as we are not testing a restaurant feature, but rather a fundamental feature. This being said, i would suggest that we remove this test from here and find a way to integrate it into the main pos tests. ( in a future task )  What do you think?

Reply:
Yeah that makes sense to me. I removed the test